examples/uefi-demo/main.rs

```diff
@@ -61,6 +79,20 @@ fn timer_freq() -> u64 {
     }
 }
 
+fn pointer_init() {
+    // mouse pointer
+    let bs = st().boot_services();
+    let handle = bs.get_handle_for_protocol::<Pointer>().expect("miss Pointer protocol");
+
+    let mut pointer =
+        bs.open_protocol_exclusive::<Pointer>(handle).expect("can't open Pointer protocol.");
+
+    pointer.reset(false).expect("Failed to reset pointer device.");
+    info!("pointer inited, mode = {:?}.", pointer.mode());
```

Comment:
do we want to keep these calls to `info!`?

Reply:
> do we want to keep these calls to `info!`?

yes, OVMF implements UEFI pointer protocol but it can't move the pointer. by serial log could see the different `pointer.mode()` print. so keep those code please.